docs/docs/notification-center/react-components.md

```diff
@@ -212,7 +212,8 @@ The `i18n` prop can accept 2 different types of values
           <li><code>tr</code> (Turkish)</li>
           <li><code>te</code> (Telugu)</li>
           <li><code>vi</code> (Vietnamese)</li>
-          
+          <li><code>bh</code> (Bhojpuri)</li>
```

Comment:
Unfortunately there is no ISO code 639-1 for Bhojpuri. And currently is the ISO code we support. We would need to hold this one.